types/src/key.rs

```diff
@@ -58,11 +67,442 @@ impl From<HashAddr> for Key {
 }
 
 /// An alias for [`Key`]s hash variant.
-pub type ContractHash = HashAddr;
+pub type ContractHashBytes = HashAddr;
```

Comment:
Does this need to be public?  I don't think we even need this alias tbh.  I'd just use `HashAddr`.  Same for `ContractPackageHashBytes` and `ContractWasmHashBytes`.

Reply:
Removed aliases and using `HashAddr` directly